client/src/app.html

```diff
@@ -10,6 +10,8 @@
     <meta name="theme-color" content="#000091" />
 
     <meta name="description" content="Catalogage des données" />
+    <link rel="icon" href="data:," />
```

Comment:
```suggestion
    <!-- Empty favicon to prevent browsers from fetching the default /favicon.ico
    until SvelteKit adds the dynamic favicon link. -->
    <link rel="icon" href="data:," />
    
```